vaadin-dialog-flow-parent/vaadin-dialog-flow/src/main/java/com/vaadin/flow/component/dialog/Dialog.java

```diff
@@ -367,6 +432,23 @@ public Registration addResizeListener(
         return addListener(DialogResizeEvent.class, listener);
     }
 
+    /**
+     * Adds a listener that is called after user finishes dragging the overlay.
+     * It is called only if dragging is enabled (see
+     * {@link Dialog#setDraggable(boolean)}).
+     * <p>
+     * Note: By default, the component will sync the top/left values after every
```

Comment:
Let's update the JavaDoc on `addResizeListener` that resizing now also automatically syncs `left` and `top`.

Reply:
Done.